io/src/csv/read_cells.cpp

```diff
@@ -60,8 +63,7 @@ read_deduplicated_cells(std::string_view filename) {
         }
     }
     if (nduplicates > 0) {
-        std::cout << "WARNING: @traccc::io::csv::read_cells: " << nduplicates
-                  << " duplicate cells found in " << filename << std::endl;
+        TRACCC_WARNING(nduplicates << " duplicate cells found in " << filename);
```

Comment:
It is not for this PR. But this probably shouldn't be a warning in the end, but rather just a debug message. Since it's something that we expect to happen with G4 simulations coming from Acts.

But since these warnings have been good progress indicators for our file reading, let's think about normalizing the messages around here, in a follow-up PR.

Reply:
Agreed!